### ISSUE TITLE
flatpak warning

### ISSUE THREAD

**Issue by alexeiz** [flathub/org.freedesktop.LinuxAudio.BaseExtension#4]
I'm getting this warning:

```
$ flatpak search anything
(flatpak search:224085): As-WARNING **: 12:40:32.205: org.freedesktop.LinuxAudio.BaseExtension was of type addon but had no extends
```

Even though org.freedesktop.LinuxAudio.BaseExtension is not installed and nothing depends on it.

**Comment by hfiguiere** [comment]
@barthalion this is related to the change you requested in the appdata. Now that it is `type=addon` people get these warning.

Not sure what's the proper fix it.

**Comment by raphaeldc** [comment]
> I'm getting this warning:
> 
> ```
> $ flatpak search anything
> (flatpak search:224085): As-WARNING **: 12:40:32.205: org.freedesktop.LinuxAudio.BaseExtension was of type addon but had no extends
> ```
> 
> Even though org.freedesktop.LinuxAudio.BaseExtension is not installed and nothing depends on it.

I'm getting the same error here, but I guess it started after I installed and removed Guitarix (`org.guitarix.Guitarix`). Since I'm not sure, it might not be related at all.

Operating System: Ubuntu 20.04.1 LTS
Kernel: Linux 5.4.0-59-generic
Flatpak 1.6.5

I've installed `org.freedesktop.LinuxAudio.BaseExtension (20.08)` and nothing happened.

**Comment by hfiguiere** [comment]
It all came because of this:

https://github.com/flathub/org.freedesktop.LinuxAudio.BaseExtension/commit/ef2e9a7745707b5f6a6fb7c67b09aae3b6475379#r45518701

**Comment by paddylandau** [comment]
This was [reported to Redhat](https://bugzilla.redhat.com/show_bug.cgi?id=1913496), but I don't think that Redhat has anything to do with it.

**Comment by hfiguiere** [comment]
Was fixed in commit 3dde70e4a0062dda6be4f3ba86c285b6eb1e97cd
